README.md

```diff
@@ -364,6 +364,7 @@ Hurray! You successfully made a contribution! 🎉
 
   - [Lishu Gupta](https://github.com/lishugupta652)
   - [Lundbyit](https://github.com/Lundbyit)
+  - [Lohit Yarra](https://github.com/lohityarra)
```

Comment:
@lohityarra can you please add your name above Lundbyit names. your name is in 367 line you can add it in the 366